visualization/app/codeCharta/ui/copyToClipboardButton/util/clipboardString.ts

```diff
@@ -0,0 +1,21 @@
+import { FileToValue } from "./getFilenamesWithHighestMetrics"
+
+function getStringLineFromItem(item: FileToValue): string {
```

Comment:
[Optional / heavily opinionated] the type makes already sure, that it will return a string, so the string in the function name is a duplicate from my point of view
```suggestion
function getLineFromItem(item: FileToValue): string {
```

Reply:
I agree with you.

Reply:
Resolved with commit [616f8d2](https://github.com/MaibornWolff/codecharta/pull/2942/commits/616f8d29759422faf9bfb7009480c795d902a03a)